jekyll/_cci2/insights-tests.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: classic-docs
+title: "Test Insights"
+short-title: "Using Test Insights"
+description: "Using Test Insights"
+version:
+- Cloud
+---
+
+## Overview
+
+Using CircleCI Test Insights, users are able to analyze their test performance
+within their most recent executions and identify opportunities for test
+optimization. To access Test Insights, select the **Tests** tab on the
+**Workflow Insights** page. 
+
+The following test insights are available:
+
+### Performance Summary
```

Comment:
JK001/header-jekyll-id-rule Header with custom id rule Expected kramdown ID attribute at next line: 11. You can fix by inserting `{: #performance-summary }` to the next line